### ISSUE TITLE
Cannot Add Source With VoiceOver on Mac

### ISSUE THREAD

**Issue by chigkim** [obsproject/obs-studio#2502]
I'm using a built-in screen reader called VoiceOver on Mac, and add source is not accessible.

## Expected Behavior

When activating add source, keyboard focus should be changed to the add source, and VoiceOver should be able to read the elements.

## Current Behavior

When click add source, VoiceOver just says OBS has a new window.
Even if you manually switch keyboard focus to the new window, it just says menu.

## Steps to Reproduce

1. Turn on VoiceOver by pressing command+f5
2. Click add for source
3. VoiceOver says OBS has a new window, and you cannot navigate the elements for the source with VoiceOver.

**Comment by WizardCM** [comment]
Seems to work alright with Narrator on Windows in OBS 26.1.0-rc2.

Could you verify if this is still an issue in 26.1.0-rc2?

**Comment by PatTheMav** [comment]
I can confirm with the Tech Preview build that when hovering/moving through the options for new scenes VoiceOver calls every Item "New Window".

**Comment by nebeker** [comment]
Reproducible, as of macOS 12.3 and 27.2.2. There is a workaround based on forcing mouse movement, similarly to #6200, but this is far from standard behavior and should not be encouraged.

**Comment by nebeker** [comment]
Seems related to an upstream Qt bug: https://bugreports.qt.io/browse/QTBUG-44853.

**Comment by Fenrirthviti** [comment]
Closing as resolved as this hasn't had updates in many years, and we have made several updates to Qt that have a good chance of resolving this issue. If this is still a problem, please open a new issue report on the current version with updated reproduction steps.
